consensus/tdpos/common.go

```diff
@@ -346,3 +346,15 @@ func (tp *TDpos) validateCheckValidater(desc *contract.TxDesc) (int64, int64, er
 	}
 	return version, term, nil
 }
+
+func (tp *TDpos) isAuthAddress(candidate string, initiator string, authRequire []string) bool {
+	if candidate == initiator {
+		return true
+	}
+	for _, value := range authRequire {
```

Comment:
Note that initiator and authRequire could be Address or Account, but candidate should be address. So here should only check the "address part" of initiator and authRequire.